util/src/main/java/com/google/cloud/hadoop/util/StatusMetrics.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2013 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Gul Jain changes
+package com.google.cloud.hadoop.util;
+
+public interface StatusMetrics {
```

Comment:
Let us set @InterfaceAudience.Private so that we can change the definition later if we wanted to.